ast.go

```diff
@@ -2597,9 +2603,19 @@ func (s *ShowMeasurementsStatement) String() string {
 	var buf strings.Builder
 	_, _ = buf.WriteString("SHOW MEASUREMENTS")
 
-	if s.Database != "" {
+	if s.Database != "" || s.WildcardDatabase {
 		_, _ = buf.WriteString(" ON ")
-		_, _ = buf.WriteString(s.Database)
+		if s.WildcardDatabase {
+			_, _ = buf.WriteString("*")
```

Comment:
I hate the fact the WriteString can return an error, but never actually does. I'm torn between wanting to check it (in case the implementation changes) or ignore it, because the error case is always dead code.